example/demo/src/app.rs

```diff
@@ -133,6 +143,8 @@ fn HomePage() -> impl IntoView {
             </div>
             <div class="relative w-full flex justify-end right-4 z-10">
                 <div class="h-full flex flex-col items-center justify-around">
+                    <button _ref=node_ref>"Click here to set k key to inc"</button>
+                    <button _ref=node_ref_disable>"Click here to disable k key"</button>
```

Comment:
We should probably refactor this, instead of two buttons, we can create an input where upon focus the keybindings should fire properly, else it shouldn't.

Reply:
But this can be addressed in a different PR

Reply:
^ holy shit can this guy stop adding extra comments and just comment in one comment box?

Reply:
> ^ i'll probably refactor the demo since I was going to anyway with the scopes

I kind of figured. I just wanted to get the functionality point across.